### PR TITLE
MLeap for local model scoring

### PR DESCRIPTION
**Pull request by tovbinm**
**Related issues**
[Aardpfark](https://github.com/CODAIT/aardpfark) is no longer supported.

**Describe the proposed solution**
Switching to [MLeap](https://github.com/combust/mleap) runtime instead.

**Describe alternatives you've considered**
N/A

### CODE REVIEW

**Comment by tovbinm** [comment]
@alexandrnikitin ^

File: local/README.md [hunk 1]
```diff
@@ -44,6 +39,6 @@ val scoreFn = new OpWorkflowRunnerLocal(workflow).score(opParams)
 Below is an example of measured scoring performance on 6m records with 10 fields and 12 transformations applied.
 Executed on MacBook Pro i7 3.5Ghz in a single thread.
 ```
-Scored 6,000,000 records in 239s
-Average time per record: 0.0399215ms
+Scored 6,000,000 records in 710s
```
Comment (gerashegalov) [inline local/README.md]: interesting that there is 3x throughput difference

Comment (tovbinm) [reply local/README.md]: I had to warm up the JVM. Updated the numbers.

Comment (tovbinm) [reply local/README.md]: It's slightly faster.

File: local/src/main/scala/com/salesforce/op/local/OpWorkflowModelLocal.scala [hunk 2]
```diff
@@ -57,29 +58,27 @@ trait OpWorkflowModelLocal extends Serializable {
    */
   implicit class RichOpWorkflowModel(model: OpWorkflowModel) {
 
-    private implicit val formats = DefaultFormats
-
     /**
-     * Internal PFA model representation
+     * Internal OP model representation
```
Comment (leahmcguire) [inline local/src/main/scala/com/salesforce/op/local/OpWorkflowModelLocal.scala]: MLeap?

**Pull request merged**